tests/remote_store/remote_store/restore.yaml

```diff
@@ -0,0 +1,26 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test restore from a remote store.
+prologues:
+  - path: /_bulk
+    method: POST
+    parameters:
+      refresh: true
+    request:
+      content_type: application/x-ndjson
+      payload:
+        - {create: {_index: books, _id: book1}}
+        - {author: Harper Lee, title: To Kill a Mockingbird, year: 60}
```

Comment:
Wow, I didn't realize this book was nearly 2000 years old.

Reply:
It's some bad copy paste from other tests that try to do "math". I updated the years where I could in this PR.